nodejs/week2/homework/meal-sharing/src/backend/api/reviews-router.js

```diff
@@ -0,0 +1,28 @@
+const express = require("express");
+const router = express.Router();
+
+const reviews = require("./../data/reviews.json");
+
+router.get("/", async (request, response) => {
+  try {
+    console.log(reviews);
+    console.log("in /api/reviews");
+  } catch (error) {
+    throw error;
+  }
+});
+router.get("/:id", async (request, response) => {
```

Comment:
GET `/api/reviews/gggggg`   think of cases where `id` is not a number and return error to the user . e.g : ` response.status(400).send('Review IDs must be numbers.');`